docs/index.md

```diff
@@ -4,8 +4,8 @@ page_title: "Provider: Elastic Cloud (EC)"
 
 # Elastic Cloud Provider
 
-The Elastic Cloud can be used to configure and manage Elastic Cloud deployments using the Elastic Cloud
-APIs. Documentation regarding the Data Sources and Resources supported by the Elastic Cloud provider can be found in the navigation to the left.
+The Elastic Cloud provider can be used to configure and manage Elastic Cloud deployments using the Elastic Cloud
```

Comment:
```suggestion
The Elastic Cloud Terraform provider can be used to configure and manage Elastic Cloud deployments using the Elastic Cloud
```

Global comment: The full product name is `Elastic Cloud Terraform provider`, based on this [gdoc overview draft](https://docs.google.com/document/d/1hqGF2KB3mCkKJeJzvNXpQriNixr62AdOgsBEkFqxkHk/edit#) from @Kushmaro. The other short docs I put together use that same name.